nixos/tests/nixos-rebuild-target-host.nix

```diff
@@ -0,0 +1,96 @@
+import ./make-test-python.nix ({ pkgs, ... }: let
+  inherit (import ./ssh-keys.nix pkgs) snakeOilPrivateKey snakeOilPublicKey;
+
+  deployerAddress = "192.168.0.1";
+  targetAddress = "192.168.0.2";
+in {
+  name = "nixos-rebuild-target-host";
+
+  nodes = {
+    deployer = { lib, ... }: {
+      imports = [ ../modules/profiles/installation-device.nix ];
+
+      networking.interfaces.eth1.ipv4.addresses = [ { address = deployerAddress; prefixLength = 24; } ];
+      networking.hosts.${targetAddress} = [ "target" ];
```

Comment:
I though this happened automatically. Can this line be removed?

Reply:
Done :+1: